### PR TITLE
test: adds guidepup test for notification

### PR DESCRIPTION
**Pull request by bruno-sch**
## Proposed changes

test: adds guidepup test for notification

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Review by nmerget — DISMISSED** [review]

File: showcases/screen-reader/tests/notification.spec.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		test,
+		title: 'default',
+		description:
+			'should have texts inline or as data-label attributes (next())',
+		url: './#/06/notification?page=density',
+		async testFn(voiceOver, nvda) {
+			if (nvda) {
+				// Skip functional notification
+				await nvda?.next();
+				await nvda?.next();
```
Comment (mfranzke) [inline showcases/screen-reader/tests/notification.spec.ts:16]:
```suggestion
```

why not remove one of these lines to end up on the same element for both screenreaders and make the snapshots a little bit more "comparable" ?

Comment (nmerget) [reply showcases/screen-reader/tests/notification.spec.ts]: fixed

**Review by nmerget — APPROVED** [review]

File: showcases/screen-reader/tests/notification.spec.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { getTest, testDefault } from '../default';
+
+const test = getTest();
+
+test.describe('DBNotification', () => {
+	testDefault({
+		test,
+		title: 'default',
+		description:
+			'should have texts inline or as data-label attributes (next())',
```
Comment (mfranzke) [inline showcases/screen-reader/tests/notification.spec.ts:10]: how we test the latter (`data-label`)?

Comment (mfranzke) [reply showcases/screen-reader/tests/notification.spec.ts:10]: https://github.com/db-ui/mono/issues/3069

**Pull request merged**